gaia-list.js

```diff
@@ -31,11 +31,21 @@ module.exports = component.register('gaia-list', {
     this.setupShadowRoot();
     this.els = { inner: this.shadowRoot.querySelector('.inner') };
     this.addEventListener('click', this.onPointerDown);
-    this.makeAccessible();
+    // put non-critical operations in post-load time frame
+    // to save loadtime.
+    setTimeout(() => { this.makeAccessible(); });
   },
 
   makeAccessible: function() {
     [].forEach.call(this.children, (el) => { el.tabIndex = 0; });
+
+    // Make sure all icons are hidden from screen reader
+    var icons = this.getElementsByTagName('i');
+    if (icons) {
```

Comment:
This conditional will always pass, even if an empty list is returned.

```js
if ([]) { ... }
```

Let's just remove lines `44` and `48`.